apps/e2e-tests/config.ts

```diff
@@ -0,0 +1,4 @@
+export const config = {
+  appOrigin: process.env.ASAP_API_URL || 'https://api-dev.hub.asap.science',
```

Comment:
We use app origin to mention the main page.
To avoid expectation mismatch I would change this to `apiOrigin`.